.github/workflows/process-fbc-fragment.yaml

```
@@ -0,0 +1,72 @@
+name: FBC Processor
+run-name: FBC Processor
+on:
+  workflow_dispatch:
+    branches:
```

actionlint (1.7.7)

expected "inputs" key for "workflow_dispatch" section but got "branches"

(syntax-check)